src/privatesend-client.cpp

```diff
@@ -1223,7 +1225,8 @@ bool CPrivateSendClient::CreateDenominated(const CompactTallyItem& tallyItem, bo
 {
     std::vector<CRecipient> vecSend;
     CAmount nValueLeft = tallyItem.nAmount;
-    nValueLeft -= PRIVATESEND_COLLATERAL; // leave some room for fees
+    CAmount nMixingCollateral = CPrivateSend::GetCollateralAmount();
```

Comment:
It would be more readable to use it as 'CPrivateSend::GetCollateralAmount()' everywhere and not assigning to non-const local variable.